src/test/java/stepDefinition/RoomsAreDisplayedSteps.java

```diff
@@ -31,33 +31,30 @@ public void userValidatesCalendarIsPresent() {
     }
 
     @When("User selects the check-in date {string} and check out date {string}")
-    public void userSelectsTheStartDateStartdateAndEndDateEnddate(String dateCheck, String dateCheck1) throws InterruptedException {
-        SeleniumActions.click(RoomPage.dateVariable(dateCheck), driver);
-        SeleniumActions.click(RoomPage.dateVariable(dateCheck1), driver);
+    public void userSelectsTheStartDateStartdateAndEndDateEnddate(String dateSelect, String dateSelect1) {
```

Comment:
call the parameters checkInDate and CheckOutDate

Reply:
Done